category/traversable/instances.lean

```diff
@@ -0,0 +1,256 @@
+/-
+Copyright (c) 2018 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Simon Hudon
+
+Instances of `traversable` for types from the core library
+-/
+
+import category.traversable.basic
+import category.basic
+import category.functor
+import category.applicative
+
+universe variables u v w
```

Comment:
Could you try to remove explicit universe annotations wherever possible? Ideally lean should be able to figure out on its own when two universe variables must be the same, and that will make sure you aren't unnecessarily restricting the universe levels. Also `universe variables` is just `universes` since lean 3.